Index: src/components/FormBuilder.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/components/FormBuilder.tsx	(date 1568025845640)
+++ src/components/FormBuilder.tsx	(date 1568025845640)
@@ -6,10 +6,8 @@
     IEntities,
     IFields, IGroups,
     ILayout,
-    ILayoutFields,
     ISchema,
     Layout,
-    LayoutFields,
     RenderOption,
     SimpleObj
 } from "../interfaces/SchemaInterfaces";
@@ -177,7 +175,7 @@
         }
     };
 
-    fieldEvaluator = (fields: Array<IFields> | Fields, layoutFields?: Array<ILayoutFields> | LayoutFields, groups?: Group) => {
+    fieldEvaluator = (fields: Array<IFields> | Fields, layoutFields?: Array<IFields> | Fields, groups?: Group) => {
         if (layoutFields) {
             if (Array.isArray(layoutFields)) {
                 // Field of type Array or Object both are handled as array in each cases, every block has its own variable fieldArray for block scoping
